x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields.tsx

```diff
@@ -76,80 +81,126 @@ const columns: Array<EuiBasicTableColumn<HighlightedFieldsTableRow>> = [
       values: string[] | null | undefined;
       scopeId: string;
       isPreview: boolean;
+      showCellActions: boolean;
     }) => (
-      <CellActions field={description.field} value={description.values}>
-        <HighlightedFieldsCell
-          values={description.values}
-          field={description.field}
-          originalField={description.originalField}
-        />
-      </CellActions>
+      <>
+        {description.showCellActions ? (
+          <CellActions field={description.field} value={description.values}>
```

Comment:
the cell actions can be made context agnostic as well, this can be done in a separate PR
```
<CellActions 
   field={description.field} 
   value={description.values}
   disabled={!description.showCellActions}
   ...>

```

Reply:
We can but don't really need to as AI for SOC does not have CellActions (for now).